Readme.md

```diff
@@ -1,16 +1,20 @@
 <!-- default badges list -->
-![](https://img.shields.io/endpoint?url=https://codecentral.devexpress.com/api/v1/VersionRange/128572485/15.2.4%2B)
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E1370)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
 
-* [Form1.cs](./CS/ConstLines/Form1.cs) (VB: [Form1.vb](./VB/ConstLines/Form1.vb))
-<!-- default file list end -->
-# How to add a constant line to an axis
+# Chart for WinForms - How to Create and Customize a Constant Line
+
+This example demonstrates how create and customize a [constant line](https://docs.devexpress.com/WindowsForms/5783/controls-and-libraries/chart-control/axes/constant-lines) at runtime.
+
+![Constant Line](image/chart.png)
 
+Since constant lines belong to a diagram's [axis](https://docs.devexpress.com/WindowsForms/5779/controls-and-libraries/chart-control/diagram/axes), it is required to cast the [diagram](https://docs.devexpress.com/WindowsForms/5778/controls-and-libraries/chart-control/diagram) object to the appropriate diagram's type. After that, you can access axes and manage their constant lines collection.
```

Comment:
Since constant lines belong to a diagram's [axis](https://docs.devexpress.com/WindowsForms/5779/controls-and-libraries/chart-control/diagram/axes), you need to cast the [diagram](https://docs.devexpress.com/WindowsForms/5778/controls-and-libraries/chart-control/diagram) object to the appropriate diagram's type. After that, you can access axes and manage their constant lines collection.